Orange/data/table.py

```diff
@@ -264,23 +264,21 @@ def from_table(cls, domain, source, row_indices=...):
 
         global _conversion_cache
 
-        def sparse_to_flat(x):
-            if sp.issparse(x):
-                x = np.ravel(x.toarray())
-            return x
-
-        def get_columns(row_indices, src_cols, n_rows, dtype=np.float64):
+        def get_columns(row_indices, src_cols, n_rows, dtype=np.float64, is_sparse=False):
```

Comment:
Not a huge deal, but: table.py mostly obey the 80-column limit. Can you break the longer lines in this PR?